app/experimenter/kinto/README.md

```diff
@@ -0,0 +1,158 @@
+
+## Remote Settings
+
+Remote Settings organizes data into buckets, which contain collections, which contain records.
+
+### Buckets
+
+For our purposes there are two buckets to consider:
+
+- `main-workspace`: a staging area where all incoming changes are written
+- `main`: the currently published set of data that clients read from
+
+All collections appear in both `main` and `main-workspace`. When changes are written to a collection in `main-workspace`, the collection may be marked for review. If the change is approved, the contents of that collection in `main-workspace` will be promoted to `main` where they can be read by clients. If the change is rejected, `main-workspace` will be reverted to its previous state before the change was made. Changes can only be reviewed, approved, rejected, or promoted for an entire collection.
+
+### Collections
+
+Experimenter interacts with three collections:
+
+- `nimbus-desktop-experiments`
+  - The collection that Firefox Desktop is configured to read from by default.
+- `nimbus-mobile-experiments`
+  - The collection that Firefox for Android (Fenix) is configured to read from by default.
+- `nimbus-preview`
+  - Any client can be manually configured to read from the `nimbus-preview` collection for local testing, QA, and validation before an experiment launches.
+
+A collection may or may not be configured to require reviews for changes.
+
+- `nimbus-desktop-experiments` and `nimbus-mobile-experiments` require reviews for all changes
+- `nimbus-preview` does not require reviews for any changes and can be modified directly by Experimenter as a single operation
+
+Reviews in Remote Settings serve multiple functions:
+
+- To ensure that erroneous data is not sent to clients which may adversely affect those clients
+- To provide an additional security boundary from preventing experiments from being deployed without being approved by multiple parties
+
+For a collection that requires reviews where multiple records are modified, there is no way to promote a change to one record from `main-workspace` to `main` without also promoting all the others. For this reason, the workflows diagrammed below enforce that only a single change can ever be made to a collection within a single review cycle.
+
+### Experiment States
+
+An experiment has two distinct states:
+
+- Its lifecycle state
+- Its publish state
```

Comment:
I feel like if you write a couple high-level sentences around here as to _why_ there's two needed distinct states and maybe mention that up until this point we only had one, we don't really need that ADR I mentioned since this doc basically covers everything 🙂